src/locales/en.json

```diff
@@ -329,5 +329,10 @@
   "USER_PROFILE_UPLOAD_MISSING_DOCUMENTS": "Upload Missing Documents",
   "WALLET_INTERFACE": "My Wallet",
   "WEBVIEW_COMPLETE_APPLICATION_TITLE": "Complete Application",
-  "WEBVIEW_FORM_UI_TITLE": "Form UI"
+  "WEBVIEW_FORM_UI_TITLE": "Form UI",
+  "ENCRYPTION_MODAL_TITLE": "Field Encryption",
```

Comment:
1. use prefix as
`ADDFIELDS_ENCRYPTION_MODAL`

2. add alphabetically

3. add constants for mr, hi

Reply:
This is done